pkg/common/synced_map_utils_test.go

```diff
@@ -26,17 +26,20 @@ import (
 )
 
 /*
-	apiVersion: mcm.ibm.com/v1
-		kind: GRCPolicy
-		metadata:
-			name: GRC-policy
-		spec:
-			namespaces:
-				include: ["default"]
-				exclude: ["kube*"]
-			remediationAction: enforce # or inform
-			conditions:
-				ownership: [ReplicaSet, Deployment, DeamonSet, ReplicationController]
+apiVersion: mcm.ibm.com/v1
+kind: GRCPolicy
+metadata:
+
+	name: GRC-policy
+
+spec:
+
+	namespaces:
+		include: ["default"]
+		exclude: ["kube*"]
+	remediationAction: enforce # or inform
+	conditions:
+		ownership: [ReplicaSet, Deployment, DeamonSet, ReplicationController]
```

Comment:
This got reformatted for the improved go docs format, I think. But we probably just want to remove the comment, since it's out of date. Doesn't need to happen in this PR, just mentioning it.

Reply:
- https://github.com/stolostron/cert-policy-controller/pull/181

Reply:
Good idea 🙂